modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -45,15 +45,16 @@ $ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal --regi
 +
 [source,terminal]
 ----
-$ gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
+$ ingress_backendservice=$(gcloud compute backend-services list --filter="backends.group~${INFRA_ID}" --format='value(name)' | grep -v "${INFRA_ID}")
 ----
+** If `ingress_backendservice` is not empty, run the following commands for the bootstrap group:
 +
 [source,terminal]
 ----
-$ gsutil rb gs://${INFRA_ID}-bootstrap-ignition
```

Comment:
This line along with the following line of `$ gcloud deployment-manager deployments delete ${INFRA_ID}-bootstrap` should not be deleted, instead the new lines should be inserted before them.